src/main/java/karen/command/ByeCommand.java

```diff
@@ -0,0 +1,20 @@
+package karen.command;
+
+import karen.Storage;
+import karen.Ui;
+
+/**
+ * To feedback Karen to exit and close the application.
+ */
+public class ByeCommand extends Command {
+    @Override
+    public boolean isExit() {
+        return true;
+    }
+
+    @Override
+    public String execute(Ui ui, Storage storage) {
+        return Ui.GOODBYE;
```

Comment:
A really succinct way to handle the 'bye' command 👍 
Would it be a good idea to also consider a case where user inputs 'bye [arg]' accidentally? 

_Perhaps the user mistakenly types 'bye' when he intended to type 'todo' with an argument 😆 in which case Karen can be helpful by reminding him/her that 'bye' takes no arguments and maybe he/she should type 'bye' again._